lambda/conftest.py

```diff
@@ -1,28 +1,29 @@
 import json
+from typing import Dict, List, Union
 
 import pytest
 
 
 @pytest.fixture()
-def sns_message():
+def sns_message() -> Dict[str, List[Dict[str, Dict[str, str]]]]:
```

Comment:
Nice schema, perhaps name this?
```python
sns_type = Dict[str, List[Dict[str, Dict[str, str]]]]
```
@akinnane We talked about mypy not being able to check the shape of nested dicts, does that apply here?

Reply:
This is fine as each member can only be of one type. The problems start to occur when you do something like:

```Dict[str, Union[int, str, Dict[str, Union[int, str, Optional[str]]]]]```